.cargo/config.toml

```diff
@@ -21,7 +21,9 @@ rustflags = [
     "-Wclippy::macro_use_imports",
     "-Wclippy::map_flatten",
     "-Wclippy::missing_enforced_import_renames",
-    "-Wclippy::mod_module_files",
+    # Disabled because it triggers for tests/foo/mod.rs which can't be replaced
+    # easily. Locally allowing it also doesn't seem to work.
```

Comment:
Is there an issue for this?

Reply:
Yeah, I think something like that would be good. Maybe not anything called `tests` but instead just a `tests` directories that is a sibling of the `Cargo.toml` for the package being checked.